ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -6,10 +6,15 @@
 - hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
-    - name: Create and provision external computes
-      ansible.builtin.include_role:
-        name: libvirt_manager
-        tasks_from: deploy_edpm_compute.yml
+  - name: Create and provision external computes
+    ansible.builtin.include_role:
+      name: libvirt_manager
+      tasks_from: deploy_edpm_compute.yml
```

Comment:
Can we call this task after "call predeploy" tasks? It will help the user who just wants to deploy podified control plane